{% extends "japps/app_description.html" %}
{% load static %}
{% block image %}
  <img img src="{% static 'japps/images/mikado-logo.png' %}" alt="[IMAGE: Mikado logo]" />
{% endblock %}
{% block description %}
  <p><strong>Mikado</strong> is a lightweight Python3 pipeline to identify the most useful or “best” set of transcripts from multiple transcript assemblies. Our approach leverages transcript assemblies generated by multiple methods to define expressed loci, assign a representative transcript and return a set of gene models that selects against transcripts that are chimeric, fragmented or with short or disrupted CDS. Loci are first defined based on overlap criteria and each transcript therein is scored based on up to 50 available metrics relating to ORF and cDNA size, relative position of the ORF within the transcript, UTR length and presence of multiple ORFs. Mikado can also utilize blast data to score transcripts based on proteins similarity and to identify and split chimeric transcripts. Optionally, junction confidence data as provided by <a href="https://portcullis.readthedocs.io/en/latest/">Portcullis</a> can be used to improve the assessment. The best-scoring transcripts are selected as the primary transcripts of their respective gene loci; additionally, Mikado can bring back other valid splice variants that are compatible with the primary isoform.<br/><br/>Mikado uses GTF or GFF files as mandatory input. Non-mandatory but highly recommended input data can be generated by obtaining a set of reliable splicing junctions with <a href="https://portcullis.readthedocs.io/en/latest/">Portcullis</a>, by locating coding ORFs on the transcripts using <a href="https://transdecoder.github.io/">Transdecoder</a>, and by obtaining homology information through <a href="https://blast.ncbi.nlm.nih.gov/blast/Blast.cgi?PROGRAM=blastx&PAGE_TYPE=BlastSearch&LINK_LOC=blasthome">BLASTX</a>.<br/><br/>Our approach is amenable to include sequences generated by de novo Illumina assemblers or reads generated from long read technologies such as Pacbio.</p>
{% endblock %}
{% block doclink %}
  <p>See the <a href="https://pachterlab.github.io/kallisto/">extended documentation</a> for details, or check out the code on <a href="https://github.com/lucventurini/mikado">github</a>.</p>
{% endblock %}
{% block ref_list %}
{% endblock %}
