{% extends "japps/app_description.html" %}
{% load static %}
{% block image %}
  <img img src="{% static 'japps/images/portcullis_logo.png' %}" alt="[IMAGE: Portcullis logo]" style="max-width: 100%"/>
{% endblock %}
{% block description %}
  <p>From the documentation:
    <blockquote><strong>Portcullis</strong> stands for PORTable CULLing of Invalid Splice junctions from pre-aligned RNA-seq data. It is known that RNAseq mapping tools generate many invalid junction predictions, particularly in deep datasets with high coverage over splice sites. In order to address this, instead for creating a new RNAseq mapper, with a focus on SJ accuracy we created a tool that takes in a BAM file generated by an RNAseq mapper of the user’s own choice (e.g. Tophat2, Gsnap, STAR2 or HISAT2) as input (i.e. it’s portable). It then, analyses and quantifies all splice junctions in the file before, filtering (culling) those which are unlikely to be genuine. Portcullis output’s junctions in a variety of formats making it suitable for downstream analysis (such as differential splicing analysis and gene modelling) without additional work.</blockquote></p>
{% endblock %}
{% block doclink %}
  <p>See the <a href="https://portcullis.readthedocs.io/en/latest/">Portcullis documentation</a> or the <a href="https://github.com/maplesond/portcullis">GitHub repository</a> for details.</p>
{% endblock %}
{% block ref_list %}
  <li>Portcullis was presented at the Genome Science 2016 conference by Daniel Mapleson: <a href="https://portcullis.readthedocs.io/en/latest/_downloads/portcullis_genomescience2016_poster.pdf">poster</a>, <a href="https://portcullis.readthedocs.io/en/latest/_downloads/portcullis_genomescience2016_slides.pdf">slides</a>.</li>
{% endblock %}
