{% extends "japps/app_description.html" %}
{% load static %}
{% block image %}
  <img img src="{% static 'japps/images/tophat.png' %}" alt="[IMAGE: TopHat logo]" style="max-width: 100%"/>
{% endblock %}
{% block description %}
  <p><strong>TopHat</strong> is a fast splice junction mapper for RNA-Seq reads. It aligns RNA-Seq reads to mammalian-sized genomes using the ultra high-throughput short read aligner Bowtie, and then analyzes the mapping results to identify splice junctions between exons. </p>
  <p>CyVerse UK provides an application to use TopHat with single reads, and one to be used for paired end reads.</p>
{% endblock %}
{% block doclink %}
  <p>See the <a href="https://ccb.jhu.edu/software/tophat/index.shtml">official website</a> for more details.</p>
{% endblock %}
{% block ref_list %}
  <li>Trapnell C, Pachter L, Salzberg SL. TopHat: discovering splice junctions with RNA-Seq. Bioinformatics doi:10.1093/bioinformatics/btp120</li>
  <li>Langmead B, Trapnell C, Pop M, Salzberg SL. Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. Genome Biology 10:R25.</li>
  <li>Kim D and Salzberg SL. TopHat-Fusion: an algorithm for discovery of novel fusion transcripts. Genome Biology 2011, 12:R72</li>
  <li>Kim D, Pertea G, Trapnell C, Pimentel H, Kelley R, Salzberg SL. TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions. . Genome Biology 2013, 14:R36</li>
{% endblock %}
